var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -31,10 +31,8 @@ def install(self, spec, prefix):
         mkdirp(prefix.bin)
 
         # Install all executable non-script files to prefix bin
-        files = [x for x in glob.glob("*")
-                 if not re.match("^.*\.sh$", x) and is_exe(x)]
-        for f in files:
-            install(f, prefix.bin)
+        jar_file = 'GenomeAnalysisTK.jar'
+        install(jar_file, prefix.bin)
```

Comment:
Can you explain the change here?

Reply:
The regex wasn't finding the jar file, so I changed it to look specifically for it.

Reply:
This fix breaks gatk@4.0: as there is no longer a GenomeAnalysisTK.jar file. It's been replaced with a wrapper script named `gatk`.